lib/plugins/aws/dev/local-lambda/runtime-wrappers/node.cjs

```diff
@@ -108,15 +108,25 @@ const importFunction = async (handlerFileAbsolutePath, handlerName) => {
   try {
     handlerFunction = require(handlerFileAbsolutePath)[handlerName]
   } catch (error) {
-    if (error.code === 'ERR_REQUIRE_ESM') {
-      handlerFunction = await import(handlerFileAbsolutePath)[handlerName]
+    // Handle ESM syntax errors
+    if (
+      // This happens when the user sets "type": "module" in their package.json
+      error.code === 'ERR_REQUIRE_ESM' ||
+      // This happens when the user does not configure the build settings in serverless.yml
+      error.message === 'Cannot use import statement outside a module'
+    ) {
+      // We can't use ServerlessError here because it is ESM and this file is CommonJS
+      throw new Error(
+        'To use ESM syntax, please remove the "type" property from your "package.json" file and configure the build settings in your serverless.yml file. For more information please refer to the "build" documentation: https://www.serverless.com/framework/docs/providers/aws/guide/building',
+      )
     }
 
     throw error
   }
 
   // Make sure the handler is a function
   if (typeof handlerFunction !== 'function') {
+    // We can't use ServerlessError here because it is ESM and this file is CommonJS
```

Comment:
We now require node 18 as a minimum version for Framework, so why don't we make this an ESM script to run everything, and then the wrapper can run both CJS and ESM?

Reply:
Good point 👍